client/internal/routemanager/systemops_bsd.go

```diff
@@ -79,6 +83,32 @@ func getRoutesFromTable() ([]netip.Prefix, error) {
 	return prefixList, nil
 }
 
+func retryFetchRIB() ([]byte, error) {
+	var out []byte
+	operation := func() error {
+		var err error
+		out, err = route.FetchRIB(syscall.AF_UNSPEC, route.RIBTypeRoute, 0)
+		if err != nil && strings.Contains(err.Error(), "sysctl: cannot allocate memory") {
```

Comment:
This whole thing was already addressed in https://go-review.googlesource.com/c/net/+/313649/3/route/route.go, but I assume sometimes it's still not sufficient?

Anyway, let's use the error here:

```suggestion
		if errors.Is(err, syscall.ENOMEM) {
```